### PR TITLE
docs: Editorial changes to the GitHub App page

### PR DESCRIPTION
**Pull request by adamdmharvey**
## Hey, I just made a Pull Request!

This was linked today and one quick read showed a minor typo, so took a quick stab at some editorial changes.

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [x] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)

### CODE REVIEW

**Comment by backstage-service** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr4549-0fnbxly9ludtmbrbexdd83ns9nnyyymf.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/602bd2ff65f8171f39f177e1

**Review by benjdlambert — APPROVED** [review]
@adamdmharvey Vale caught you! But looks good otherwise! Thanks! 👍

File: docs/plugins/github-apps.md [hunk 1]
```diff
@@ -46,9 +46,9 @@ You have to create the GitHub Application manually using these
 as GitHub Enterprise does not support creation of apps from manifests.
 
 Once the application is created you have to generate a private key for the
-application it in a `yaml` file.
+application and place it in in a YAML file.
```
Comment (benjdlambert) [inline docs/plugins/github-apps.md:49]: Think this needs fixing too @adamdmharvey

Comment (adamdmharvey) [reply docs/plugins/github-apps.md]: thx! 😁

**Comment by backstage-service** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr4549-0fnbxly9ludtmbrbexdd83ns9nnyyymf.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/602bd2ff65f8171f39f177e1

**Pull request merged**